.circleci/config.yml

```diff
@@ -60,7 +68,8 @@ executors:
         default: '10'
     docker:
       - image: circleci/node:<< parameters.tag >>
-
+    environment:
+      DEBUG: lerna
```

Comment:
Good question. I'm happy to remove it, but I've found this immensely useful locally.  In fact, I've had it set globally in my environment for the last forever because it's output is so useful.  Granted, most of that usefulness was to have better clarity into various circumstantial failures which would be less likely in a controlled environment like CircleCI (e.g. dirty Git work tree).

I propose we leave it for now, but possibly remove it in a couple weeks after this workflow validates itself!